testing/xfpga/test_umsg_c.cpp

```diff
@@ -194,6 +226,47 @@ TEST_P(umsg_c_p, set_umsg_attr_ioctl_err) {
   system_->register_ioctl_handler(FPGA_PORT_GET_INFO, dummy_ioctl<-1,ENOTSUP>);
   EXPECT_EQ(FPGA_INVALID_PARAM, xfpga_fpgaSetUmsgAttributes(handle_, value));
 }
+
+
+TEST_P(umsg_c_p, get_umsg_ptr_ioctl_err) {
+  uint64_t *value = 0;
+
+  // register an ioctl handler that will return -1 and set errno to EINVAL
+  system_->register_ioctl_handler(FPGA_PORT_UMSG_ENABLE, dummy_ioctl<-1,EINVAL>);
+  system_->register_ioctl_handler(FPGA_PORT_DMA_UNMAP, dummy_ioctl<-1,EINVAL>);
+  EXPECT_EQ(FPGA_INVALID_PARAM, xfpga_fpgaGetUmsgPtr(handle_, &value));
```

Comment:
Are two ioctl calls done inside this function?

Reply:
From what I can tell through gdb, two+ ioctl calls can happen through calling fpgaGetUmsgPtr